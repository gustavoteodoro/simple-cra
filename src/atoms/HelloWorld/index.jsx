import React from 'react';

import {
  HelloWorldContainer,
  HelloWorldText,
} from './styles';

import {
  hello,
} from './data.json';

function HelloWorld() {
  return (
    <HelloWorldContainer>
      <HelloWorldText>{hello}</HelloWorldText>
    </HelloWorldContainer>
  );
}

export default HelloWorld;
