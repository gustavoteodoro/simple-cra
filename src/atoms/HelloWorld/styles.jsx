import styled from 'styled-components';

export const HelloWorldContainer = styled.div`
`;

export const HelloWorldText = styled.h1`
`;
