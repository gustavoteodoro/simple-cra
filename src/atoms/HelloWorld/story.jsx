import React from 'react';
import { storiesOf } from '@storybook/react';

import HelloWorld from './index';

storiesOf('Atoms/HelloWorld', module)
  .add('Default', () => (
    <HelloWorld />
  ));
