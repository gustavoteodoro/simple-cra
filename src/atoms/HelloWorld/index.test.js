import React from 'react';
import renderer from 'react-test-renderer';

import HelloWorld from './index';

it('HelloWorld: renders correctly', () => {
  const tree = renderer
    .create(<HelloWorld />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
