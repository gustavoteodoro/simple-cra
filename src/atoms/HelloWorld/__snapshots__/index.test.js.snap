// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld: renders correctly 1`] = `
<div
  className="sc-bdVaJa fePDys"
>
  <h1
    className="sc-bwzfXH bFHdpU"
  >
    Hello, dear world!
  </h1>
</div>
`;
